app/database/default/models/global.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Model} from '@nozbe/watermelondb';
+import {MM_TABLES} from '@constants/database';
+import field from '@nozbe/watermelondb/decorators/field';
+import json from '@nozbe/watermelondb/decorators/json';
+
+export default class Global extends Model {
+    static table = MM_TABLES.DEFAULT.GLOBAL
+
+    @field('name') name!: string
+    @json('value', (rawJson) => rawJson) value!: string[]
```

Comment:
Just curious, can't we use a TS definition here for the sanitizer?

Reply:
Yes we can add a definition to the sanitizer.  

I thought of leaving it for now until we confirm the final 'shape' of the return value.

The reason being that the sanitizer gets the return value of JSON.parse function.  This can be either an Object, an array or a primitive value.  Then in the sanitizer body, we would do further transformation on that 'rawJson' to make it fit a specific shape.

I can add a TODO comment next to each @json field for now.  What do you think?